components/ui_helpers/lib/ui_helpers/workflow_helper.rb

```diff
@@ -130,12 +130,25 @@ def translation_placeholder_text(text)
     # set YAML text placeholders
     def set_text_placeholders(text) # rubocop:disable Naming/AccessorMethodName
       return "" if text.nil?
+      text.gsub! '<filing-as-head-placeholder>', l10n('faa.filing_as_head_of_household')
       # set application applicable year placeholder
       if text.include? '<application-applicable-year-placeholder>'
         text.sub! '<application-applicable-year-placeholder>', FinancialAssistanceRegistry[:enrollment_dates].setting(:application_year).item.constantize.new.call.value!.to_s
       else
         text
       end
     end
+
+    def step_hidden?(line)
+      if line.cells[1]&.attribute == "is_filing_as_head_of_household"
+        return FinancialAssistanceRegistry.feature_enabled?(:filing_as_head_of_household) ? "hide filing-as-head-of-household" : "hide"
```

Comment:
This may also cause a submission issue please check when the settings are set to false.